packages/addons/addon-depends/faad2/package.mk

```diff
@@ -0,0 +1,47 @@
+################################################################################
```

Comment:
For original work we'll use LE headers, but this package and dependencies are based on existing OE packages so we keep the original headers. We have no issues respecting/acknowledging the original authors.